core/src/main/java/io/opencensus/stats/Measure.java

```diff
@@ -66,6 +69,11 @@ private Measure() {
 
     /**
      * Constructs a new {@link MeasureDouble}.
+     *
+     * @param name name of {@code Measure}. Suggested format: {@code <web_host>/<path>}.
+     * @param description description of {@code Measure}.
+     * @param unit unit of {@code Measure}.
+     * @return a {@code MeasureDouble}.
      */
     public static MeasureDouble create(String name, String description, String unit) {
       // TODO(dpo): ensure that measure names are unique, and consider if there should be any
```

Comment:
This method should validate the String, using StringUtil.  You could also remove part of the TODO.